app/src/main/java/com/bunbeauty/papakarlo/feature/productdetails/ProductDetailsFragment.kt

```diff
@@ -83,6 +99,10 @@ class ProductDetailsFragment :
             ProductDetailsState.Event.NavigateBack -> findNavController().popBackStack()
             ProductDetailsState.Event.NavigateToConsumerCart -> findNavController()
                 .navigateSafe(globalConsumerCartFragment())
+
+            // set result?
```

Comment:
Remove extra comment

Reply:
need sync
will we return something afted adding produc?